indra/resources/default_config.json

```diff
@@ -0,0 +1,26 @@
+{
+    // Path to the REACH reading system's fat JAR
+    "REACHPATH": "",
+    // Version of REACH
+    "REACH_VERSION": "",
+    // Path to the EIDOS reading system's fat JAR
+    "EIDOSPATH": "",
+    // Path to the Sparser reading system folder
+    "SPARSERPATH": "",
+    // The default memory limit for the Java/Scala interface
+    // (BioPAX, REACH, EIDOS)
+    "INDRA_DEFAULT_JAVA_MEM_LIMIT": "8g",
+    // API keys to access Elsevier content
+    "ELSEVIER_API_KEY": "",
+    "ELSEVIER_INST_KEY": "",
+    // API keys to access BioGRID content
+    "BIOGRID_API_KEY": "",
+    // Credendials for ndexbio.org
+    "NDEX_USERNAME": "",
+    "NDEX_PASSWORD": "",
+    // Addresses to INDRA database instances
+    "primary": "",
+    "test-1": "",
+    "test-2": "sqlite:///indra_test.db"
+}
+
```

Comment:
I just added this to delete actual values from the config file, and to make the point that we may want a format in which comments are allowed (JSON doesn't actually allow commenting). Perhaps configparser (https://docs.python.org/3/library/configparser.html) or YAML could work. Also, the keys for the database-related settings are too generic this way (primary, test-1, test-2), they could have more descriptive names.